single-build-desktop.bat

```diff
@@ -71,7 +71,6 @@ call :testEnv QT_PATH
 call :testEnv QT_BIN_PATH
 call :testEnv VCINSTALLDIR
 call :testEnv Png2Ico_EXECUTABLE
-call :testEnv QTKEYCHAIN_CMAKE_DIR
```

Comment:
Shouldn't that variable be cleaned up in build-desktop.bat then?

Reply:
Yes you are right.. I missed that